ts/store/reducers/entities/services/index.ts

```diff
@@ -225,4 +227,25 @@ export const notSelectedServicesSectionsSelector = createSelector(
   }
 );
 
+// Get the number of visible services that are not yet marked as read
```

Comment:
Where are you checking the read state?

Reply:
in this commit  https://github.com/teamdigitale/io-app/pull/1336/commits/2131cbba7eb43b67c2428d4d92304cb1564e4641 I added the state read check